pkg/storage/stores/indexshipper/downloads/table_manager.go

```diff
@@ -349,6 +362,7 @@ func (tm *tableManager) ensureQueryReadiness(ctx context.Context) error {
 		level.Info(tm.logger).Log(
 			"msg", "index pre-download for query readiness completed",
 			"users_len", len(usersToBeQueryReadyFor),
+			"users", strings.Join(usersToBeQueryReadyFor, ","),
```

Comment:
isn't the final log line going to be too long? you'd be logging it for every queryReadiness call (the log level is info)